azurerm/internal/services/dns/dns_zone_resource.go

```diff
@@ -61,13 +63,86 @@ func resourceArmDnsZone() *schema.Resource {
 				Set:      schema.HashString,
 			},
 
+			"soa_record": {
+				Type:     schema.TypeList,
+				MaxItems: 1,
+				Optional: true,
+				Computed: true,
+				ForceNew: true,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"email": {
+							Type:         schema.TypeString,
+							Required:     true,
+							ValidateFunc: validate.DnsZoneSOARecordEmail,
+						},
```

Comment:
Are all these properties updatable? or should they be force new too?

Reply:
They're updatable. So I think they shouldn't be force new.